### DISCUSSION TITLE
What's the status of V5 ?

### DISCUSSION THREAD

**Discussion by matt-odk** [statelyai/xstate#4155] [General] [answered]
Hi there, it's been some time since the beta version of V5 was made available to the public last May.

Could you possibly provide some information on when we can expect the official release and when it will be ready for production use?

I've been using it for a few side projects and have been eagerly awaiting the official release of V5 before starting on company projects.

My search brought me to this link below
https://feedback.stately.ai/

**Comment by davidkpiano** [comment, marked as answer]
Hey Matt! We're still hard at work on v5 beta. Our biggest remaining tasks revolve around improving the TS types, as well as supporting V5 in the rest of the XState tooling (Stately Studio, inspector, framework adapters, etc.)

When those big tasks are done, we can move to releasing RCs.

**Reply by matt-odk** [reply]
Great, thanks for the update. Are there specific deadlines for these major tasks, or is it more of a 'when they're ready' scenario?

**Reply by davidkpiano** [reply]
It's whenever they're ready (cc. @Andarist if you have more details/estimates)
